README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/Gizra/og.svg?branch=7.x-2.x)](https://travis-ci.org/Gizra/og)
+[![Build Status](https://travis-ci.org/Gizra/og.svg?branch=7.x-2.x)](https://travis-ci.org/Gizra/og/branches)
```

Comment:
Why is this change needed?

Reply:
This change is needed because on the 7.x-2.x branch clicking that link takes us to the 8.x build results.  It should go to the 7.x-2.x build results but all I could find is /branches which at least shows all the build results for all the branches (which is better than just the latest default branch result which is what is used prior to this change)

Reply:
However this change isn't critical to the pull request, only the change to og.test is critical.  This change is merely an unrelated small improvement imho.  It's important that the og.test gets fixed so I'm willing to drop this change to the README.md if you disagree with it.